### PR TITLE
add nsq-exporter

### PR DESCRIPTION
**Pull request by 0x46616c6b**

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ hosted outside of the Prometheus GitHub organization.
    * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
    * [SMTP/Maildir MDA blackbox prober](https://github.com/cherti/mailexporter)
    * [SQL query result set metrics exporter](https://github.com/chop-dbhi/prometheus-sql)
+   * [NSQ exporter](https://github.com/lovoo/nsq_exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Can you keep the list sorted please?

**Comment by matthiasr** [comment]
I had to google NSQ to find out what it is. Would you mind linking it from the exporter's README so it's easier to understand what it is about?

**Comment by 0x46616c6b** [comment]
I will update the README ;-) I use this in my daily work, so its clear. Good point, thanks!

**Comment by brian-brazil** [comment]
You should also have a read through https://docs.google.com/document/d/1JapuiRbp-XoyECgl2lPdxITrhm5IyCUq9iA_h6jp3OY/edit to help ensure your exporter is as useful as possible.

**Pull request merged**